Mathlib/NumberTheory/FLT/Three.lean

```diff
@@ -38,3 +42,135 @@ theorem fermatLastTheoremThree_case_1 {a b c : ℕ} (hdvd : ¬ 3 ∣ a * b * c)
   rcases cube_of_not_dvd hdvd.1.2 with hb | hb <;>
   rcases cube_of_not_dvd hdvd.2 with hc | hc <;>
   rw [ha, hb, hc] <;> decide
+
+/-- To prove `FermatLastTheoremFor 3`, we may assume that `3 ∣ c`. -/
+theorem fermatLastTheoremThree_of_three_dvd_c
```

Comment:
Is this result used anywhere? If not, it can be removed, as it is subsumed by the stronger result `fermatLastTheoremThree_of_three_dvd_only_c` below.